### PR TITLE
Add recipe for player259/graphql-bundle

### PR DESCRIPTION
**Pull request by player259**
| Q             | A
| ------------- | ---
| License       | MIT

Add recipe for player259/graphql-bundle

### CODE REVIEW

File: player259/graphql-bundle/1.0/config/routes/player259_graphql.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Read the documentation: https://github.com/player259/graphql-bundle/blob/master/Resources/doc/index.rst#installation
+# Resource file contains single /graphql action
+player259_graphql:
+    resource: '@Player259GraphQLBundle/Resources/config/routing.xml'
+    prefix: /
+
+# To assign endpoint to specific url
```
Comment (OskarStark) [inline player259/graphql-bundle/1.0/config/routes/player259_graphql.yaml:7]:
```suggestion
# To assign an endpoint to a specific url
```

File: player259/graphql-bundle/1.0/config/packages/player259_graphql.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Read the documentation: https://github.com/player259/graphql-bundle/blob/master/Resources/doc/index.rst#configuration
+#player259_graphql:
+#    debug: '%kernel.debug%'
+#    logger: '?logger'
```
Comment (Nyholm) [inline player259/graphql-bundle/1.0/config/packages/player259_graphql.yaml:4]: This file is not needed.

File: player259/graphql-bundle/1.0/config/routes/player259_graphql.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Read the documentation: https://github.com/player259/graphql-bundle/blob/master/Resources/doc/index.rst#installation
+# Resource file contains single /graphql action
+player259_graphql:
+    resource: '@Player259GraphQLBundle/Resources/config/routing.xml'
+    prefix: /
+
+# To assign an endpoint to a specific url
+#player259_graphql_index:
```
Comment (Nyholm) [inline player259/graphql-bundle/1.0/config/routes/player259_graphql.yaml:8]: This is documentation. Let's remove it from the route configuration.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Please rebase your PR so we get the new CI jobs.

**Review by Nyholm — APPROVED** [review]
Awesome. Thank you

**Pull request merged**